EnglishFilter/sections/antiadblock.txt

```diff
@@ -1000,6 +1000,7 @@ game3rb.com#%#//scriptlet("abort-current-inline-script", "loadScript", ")]['inne
 ! https://github.com/AdguardTeam/AdguardFilters/issues/81107
 kittycatcam.com,morganhillwebcam.com,njwildlifecam.com,atlanticcitywebcam.com,palmbeachinletwebcam.com,juneauharborwebcam.com,portnywebcam.com,nyharborwebcam.com,portbermudawebcam.com,pompanobeachcam.com,fllbeachcam.com,ftlauderdalebeachcam.com,keywestharborwebcam.com,porttampawebcam.com,portcanaveralwebcam.com,sxmislandcam.com,mahobeachcam.com,portstmaartenwebcam.com,paradiseislandcam.com,portnassauwebcam.com,miamiairportcam.com,portmiamiwebcam.com,ftlauderdalewebcam.com,portevergladeswebcam.com#%#//scriptlet("prevent-setTimeout", "contentCheck")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/81379
+! Probably it will be possible to change AG_defineProperty rule to scriptlet when this issue will be fixed - https://github.com/AdguardTeam/Scriptlets/issues/65
```

Comment:
better add `! TODO:` to comment. In the future, it will be easier to find.

Reply:
Okay, done.